deposit/bytecode-verification/lemmas.k

```diff
@@ -0,0 +1,672 @@
+requires "edsl.k"
```

Comment:
Keeping multiple versions of lemmas.k/lemmas.md is going to be hard to maintain. Can you use the common lemmas.md and import all the rest from verification.k? Or else, maybe we should discuss what's the difference between new lemmas structure and old one. Maybe you want to use lemmas.md for old projects only. I have no way to know.

Reply:
As discussed offline, this module is written for a better organization of lemmas, facilitating an external review, at the cost of duplication (of lemmas from `lemmas.md` and `evm-data-map-symbolic.k`). See the comment in the file.